src/DynamoCore/Library/TypedParameter.cs

```diff
@@ -20,6 +21,8 @@ public TypedParameter(string parameter, ProtoCore.Type type, AssociativeNode def
             Name = parameter;
             Type = type;
             DefaultValue = defaultValue;
+
+            defaultValueString = shortArgumentName;
```

Comment:
Yes, at this point `defaultArgumentNode` is the AST that is created from the default argument expression written in the attribute as is. So the short name is effectively the same as the name written by the ZT developer in the node attribute.